docs/source-pytorch/starter/introduction.rst

```diff
@@ -99,7 +99,7 @@ For `conda <https://anaconda.org/conda-forge/pytorch-lightning>`_ users
       </div>
    </div>
 
-Or read the `advanced install guide <installation.html>`_
+Or read the `advanced install guide <https://pytorch-lightning.readthedocs.io/en/stable/starter/installation.html>`_
```

Comment:
Hard linking using the full url should be avoided whenever possible. We want that the links get generated. This applies to other places in this PR as well. Exceptions are in files that don't get generated by the docs (e.g., Readme files), or external urls to other websites.

In this specific example here, let's refer to installation.rst so the url gets properly generated.

Reply:
This raises a false positive error, I can do what you have suggested and include it in `linkcheck_ignore`.

Reply:
Yes, let's just ignore these false positives (that's one example why we didn't want a CI job that fails :))

Reply:
Resolved. Reset them and added them to `linkcheck_ingore`.